src/Aspire.Hosting.MySql/PhpMyAdminConfigWriterHook.cs

```diff
@@ -6,8 +6,10 @@
 
 namespace Aspire.Hosting.MySql;
 
-internal sealed class PhpMyAdminConfigWriterHook : IDistributedApplicationLifecycleHook
+/// <inheritdoc/>
+public class PhpMyAdminConfigWriterHook : IDistributedApplicationLifecycleHook
```

Comment:
We shouldn't be making things public for this.

https://github.com/dotnet/aspire/pull/3839#discussion_r1579781437

>> Is it worth creating a separate PR to fix this?

> If it is simple, like the case here where it is just trying to reuse constant strings, sure.

Reply:
Yes OK. I canceled it